ex3-thread-queue/main.py

```diff
@@ -0,0 +1,49 @@
+import threading
```

Comment:
Same as ex2

Function docs
And code quality checkers with the libraries I mentioned